numpy/core/tests/test_item_selection.py

```diff
@@ -70,17 +70,19 @@ def test_unicode_mode(self):
     def test_empty_partition(self):
         # In reference to github issue #6530
         a_original = np.array([0, 2, 4, 6, 8, 10])
-        a = a_original.copy()
+        for kind in ('introselect', 'floydselect'):
+            a = a_original.copy()
 
-        # An empty partition should be a successful no-op
-        a.partition(np.array([], dtype=np.int16))
+            # An empty partition should be a successful no-op
+            a.partition(np.array([], dtype=np.int16), kind=kind)
 
-        assert_array_equal(a, a_original)
+            assert_array_equal(a, a_original)
 
     def test_empty_argpartition(self):
```

Comment:
Instead of putting `for kind in ...`, you get better pytest diagnostics with
```suggestion
    @pytest.mark.parametrize('kind', ['introselect', 'floydselect'])
    def test_empty_argpartition(self, kind):
```